### PR TITLE
Add getImgCheck in ImageDao

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

File: server_api.py [hunk 1]
```diff
@@ -776,17 +776,15 @@ def edit_image_data(json_obj):
...
+                if result[0][0] != user_id and user_level < user_level_high_bound:
                     return_msg["error"] = "can not modify other user image "
                     return return_msg
-                img_type_id = pure_result[0][1]
+                img_type_id = result[0][1]
             except:
-                db.close()
-                return_msg["error"] = "no such image id : " + img_id
+                return_msg["error"] = "no such image id : {img_id}".format(img_id=img_id)
                 return return_msg
```
Comment (icecat2012) [inline server_api.py]: Another chance to format the code!
Please help us remove new line in line 789.
Make sure that only one new line between code blocks
thanks!

File: dataAccessObjects.py [hunk 2]
```diff
@@ -237,6 +237,11 @@ def addLikeCount(self,targetId):
         sql = 'UPDATE image_data SET img_like_count=img_like_count+1 WHERE img_id="{targetId}"'.format(targetId=str(targetId))
         ret = self.db.cmd(sql)
 
+    def getImgCheck(self,imgId):
+        sql = 'SELECT user_id, type_id FROM image_data WHERE img_id="{imgId}"'.format(imgId=imgId)
+        ret = db.query(sql)
```
Comment (Billy4195) [inline dataAccessObjects.py]: Using the following code is better
```
if len(ret) and len(ret[0]) == 2:
    imgInfo = dict(userId=ret[0][0],typeId=ret[0][1])
    return imgInfo
else:
    return None
```
And change the code in `server_api.py` ( don't forget to check the `imgInfo` is not `None` )

**Comment by stvreumi** [comment]
OK, finish modification.

File: dataAccessObjects.py [hunk 3]
```diff
@@ -244,6 +244,15 @@ def generateNewId(self):
             return "imge{0:010d}".format(int(res[0][0][4:]) + 1)
         else:
             return "imge0000000001"
+    def getImgCheck(self,imgId):
```
Comment (Billy4195) [inline dataAccessObjects.py]: Please change the function name, a good function name should describe its behavior.
We can probably know what this function will do when take a glance at the function name

**Comment by stvreumi** [comment]
OK, finish modification.

**Pull request merged**